tests/unit/test_regridding_service.py

```diff
@@ -205,9 +207,10 @@ def test_copy_1d_dimension_variables(self):
         dim_var_names = {'/lon', '/lat'}
         expected_attributes = {'long_name', 'standard_name', 'units'}
         vars_copied = []
-        with Dataset(self.test_1D_dimensions_ncfile, mode='r') as source_ds, Dataset(
-            target_file, mode='w'
-        ) as target_ds:
+        with (
+            Dataset(self.test_1D_dimensions_ncfile, mode='r') as source_ds,
+            Dataset(target_file, mode='w') as target_ds,
```

Comment:
I think this management of two things in a context manager is perhaps the nicest single formatting change from the update to the new formatting rules - this is definitely more readable that the previous version that was deemed "correct".

Reply:
thanks ruff